packages/vite-plugin-svelte/src/utils/compile.js

```diff
@@ -203,7 +205,7 @@ function buildMakeHot(options) {
 		// @ts-ignore
 		const adapter = options?.hot?.adapter;
 		return createMakeHot({
-			walk,
+			walk: svelte.walk,
```

Comment:
Should we also put a TODO here like in the v2 branch:

https://github.com/sveltejs/vite-plugin-svelte/blob/ec4c530125b446e1d8b5f5f84e62aafee6275320/packages/vite-plugin-svelte/src/utils/compile.js#L191-L192

Reply:
no, this won't ever be used for 5, hmr is going to work differently, likely with a compiler option.